demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -268,6 +287,15 @@ def convert_contribution_to_pack(self, files_to_source_mapping: Dict = None):
                     self.content_item_to_package_format(
                         pack_subdir, del_unified=True, source_mapping=files_to_source_mapping
```

Comment:
Instead of adding the overhead of re-unifying the package files after we've split them, why not just do this instead?
```suggestion
                        pack_subdir, del_unified=False, source_mapping=files_to_source_mapping
```